.github/workflows/lint-frontend.yaml

```diff
@@ -27,7 +27,7 @@ jobs:
       # See .stylelintignore for files that are not linted.
       - name: Run stylelint
         run: >
-          npx stylelint bookwyrm/static/css/*.css \
+          npx stylelint bookwyrm/sass/*.scss \
```

Comment:
Low importance: instead of naming the directory after the file format (`bookwyrm/sass`), I think it might be more relevant to name it after its function (`bookwyrm/style` or `bookwyrm/stylesheets`). That way if we change the compilation process (e.g. if Sass isn't supported anymore), the dir name won't have to change.